bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALPtoTTbar/ALP_ttbar_signal/ALP_ttbar_signal_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model ALP_linear_UFO_WIDTH
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = g u c d s b u~ c~ d~ s~ b~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > t t~ NP==2
```

Comment:
Does NP==2 work?
Should not it be NP=2 instead?

Reply:
Yes, this works. I did extensive studies on this.
If you want, you can have a look at my public area on lxplus: /afs/cern.ch/user/h/hjabusch/public/ALPToTTbar_genproduction
There you can find the gridpacks, log files, customizecards and also the full output directories from madgraph.